### PR TITLE
update script based on utensil's suggestion

### PR DESCRIPTION
**Pull request by JiechengZhao**
Some script change based on @utensil 's suggestion.

### CODE REVIEW

File: .i18n/zh/Game.po [hunk 1]
```diff
@@ -1210,7 +1209,7 @@ msgstr ""
 "\n"
 "【Boss战音乐】\n"
 "\n"
-"查看您的库存以查看您拥有的可用定理。\n"
+"查看您的清单以查看您拥有的可用定理。\n"
```
[the remark this answers is not in the document]
Comment (JiechengZhao) [reply .i18n/zh/Game.po:1212]: I am considering the word "工具箱“. I will try the translation in the game this weekend.

**Comment by joneugster** [comment]
Why are there changes to the `en/Game.pot` template? I think that one should stay as generated by `lake exe i18n -t`.

Otherwise, is this ready to be merged?

**Comment by JiechengZhao** [comment]
> Why are there changes to the `en/Game.pot` template? I think that one should stay as generated by `lake exe i18n -t`. (or if there are issues with the generated one, let's fix them at [lean-i18n](https://github.com/hhu-adam/lean-i18n))
> 
> Otherwise, is this ready to be merged?

It's a mistake, I will revert the `.pot` here. But there is a bug in the pot generation and this triggered Poedit to auto fix it.

gettext tool chain do not accept duplicated msgid.
```
jiecheng@jiecheng-Vostro-3420:~/workspace/lean/lean4games/NNG4$ msgmerge -U .i18n/zh/Game.po .i18n/en/Game.pot
.i18n/en/Game.pot:372: duplicate message definition...
.i18n/en/Game.pot:332: ...this is the location of the first definition
.i18n/en/Game.pot:503: duplicate message definition...
.i18n/en/Game.pot:473: ...this is the location of the first definition
.i18n/en/Game.pot:600: duplicate message definition...
.i18n/en/Game.pot:565: ...this is the location of the first definition
.i18n/en/Game.pot:1715: duplicate message definition...
.i18n/en/Game.pot:1661: ...this is the location of the first definition
.i18n/en/Game.pot:2139: duplicate message definition...
.i18n/en/Game.pot:2105: ...this is the location of the first definition
.i18n/en/Game.pot:2421: duplicate message definition...
.i18n/en/Game.pot:1911: ...this is the location of the first definition
msgmerge: found 6 fatal errors
```

**Comment by JiechengZhao** [comment]
> Why are there changes to the `en/Game.pot` template? I think that one should stay as generated by `lake exe i18n -t`. (or if there are issues with the generated one, let's fix them at [lean-i18n](https://github.com/hhu-adam/lean-i18n))
> 
> Otherwise, is this ready to be merged?

It ready to merge, all keys are translated. 

But I am still testing the game to see if something is wrong.

**Comment by joneugster** [comment]
Thanks!

**Pull request merged**